src/initializeApp.js

```diff
@@ -45,9 +45,10 @@ export default function init(urlParams, env = process.env) {
   setOptions(urlParams, store)
 
   store.dispatch(`loadLocalPreferences`)
-  store.dispatch(`checkForPersistedSession`)
+  store.dispatch(`checkForPersistedNetwork`).then(() => {
+    store.dispatch(`checkForPersistedSession`)
```

Comment:
perhaps these should be combined? 

network, session, addresses are all the same now (kind of)?

Reply:
hmmm it is not right? you can have a "last network" and or every network you have individually a "last used address"

Reply:
ah i forgot that you can have a selected network without an address. nice.